docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md

```diff
@@ -213,14 +212,8 @@ Once the release task has completed, it may be ideal to manually test the site o
 by running it locally (e.g. `http://localhost/release-3.3.4`) to do some smoke-testing
 and make sure that there are no obvious issues missed.
 
-Since `cow` will only run the unit test suite, you'll need to check
-the build status of Behat end-to-end tests manually on travis-ci.org.
```

Comment:
Well, at the moment there's no way to push to travis without publishing the tag, so this is misleading

Reply:
Maybe we mention that this would be checking the build status of the latest branch